functions-and-scope.js

```diff
@@ -16,6 +16,16 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+console.log("-----Opdracht 1a-----");
+
+let count = 0;
+for (let i=0; i<grades.length; i++) {
+  if(grades[i] >= 8) {
+    count += 1;
```

Comment:
Hij kan ietsjes simpeler 😉 
```suggestion
    count++;
```